files/templates/scrapeconfigs/additional-scrape-configs.template.yaml

```diff
@@ -530,21 +530,26 @@
   - source_labels: [__meta_kubernetes_service_labelpresent_operated_prometheus]
     regex: (true)
     action: drop
-  - source_labels: [__meta_kubernetes_service_annotationpresent_giantswarm_io_monitoring_port]
-    regex: (true)
-    action: keep
-  - source_labels: [__meta_kubernetes_service_annotationpresent_giantswarm_io_monitoring_path]
-    regex: (true)
-    action: keep
```

Comment:
We should probably keep those in under not to catch other unwanted targets.

Reply:
Well, removing those is the main objective of this PR as we want to use defaults if those annotations are not present

Reply:
If we have both __meta_kubernetes_service_annotationpresent_giantswarm_io_monitoring_path and __meta_kubernetes_service_annotationpresent_giantswarm_io_monitoring_port this means we are not using defaults :D

Reply:
Ok, but I believe those labels with `keep` action are the ones filtering out the unwanted targets.

Reply:
Yes they are but this means we have to define those annotations even when the values behind it are defaults (metrics and main port).


Maybe we can discuss it a little during planning as this is not a priority but I was thinking we could add an annotation or label to managed apps to know if the app is managed and split the targets in 2 groups (managed-apps and workloads) but we can keep this PR open for when we have time to work on this

Reply:
The monitoring of apps document is then invalid as this explains that the default values are optional when we actually require them.